dxr/templates/nunjucks/promoted.html

```diff
@@ -0,0 +1,27 @@
+{% macro promoted_list(query, promoted, promoted_count, www_root, tree) -%}
+  {% for result in promoted %}
+    <div class="result-head">
+      <div class="left-column">
+        <div class="icon-container {{ result.iconClass }}"></div>
+      </div>
+      <div>
+        {{ result.pathLine | safe }}
```

Comment:
|safe scares the heck out of me; it's been the source of a lot of security holes and is basically against Moz's code guidelines now. Can we instead wrap this in jinja.Markup() at some point in the Python code?